src/worker/builder.rs

```diff
@@ -7,13 +7,17 @@ use std::future::Future;
 use tokio::io::{AsyncRead, AsyncWrite, BufStream};
 use tokio::net::TcpStream as TokioStream;
 
+pub(crate) const GRACEFUL_SHUTDOWN_PERIOD_MILLIS: u64 = 5_000;
+
 /// Convenience wrapper for building a Faktory worker.
 ///
 /// See the [`Worker`] documentation for details.
 pub struct WorkerBuilder<E> {
     opts: ClientOptions,
     workers_count: usize,
     callbacks: CallbacksRegistry<E>,
+    shutdown_timeout: u64,
```

Comment:
Let's make this be a `Duration` and extract the millis when needed.

Reply:
Done!